Paper.tex

```diff
@@ -1365,10 +1404,27 @@ \section{Block Finalisation} \label{ch:finalisation}
 The process of finalising a block involves two stages:
 
 \begin{enumerate}
+\item executing withdrawals;
```

Comment:
My understanding is that `validate transactions` precede  `executing withdrawals`

Reply:
That's a good question! The EIP only states that it should happen after transaction *execution* but it says nothing about where it should happen during block finalization.
The transaction *validation* step purpose in the YP is simply meant to set the block header's gas used value ($B_{Hg}$), since withdrawals incur no gas cost, it has no impact if its run before or after withdrawal execution.